### PR TITLE
Add a log for starting to load cache and fix MB->MiB

### PR DESCRIPTION
**Pull request by KimKyungup**
## Proposed changes

This PR added a log for starting to load cache and fixed MB->MiB.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [ ] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [ ] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [ ] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: storage/statedb/cache_fastcache.go [hunk 1]
```diff
@@ -41,23 +43,27 @@ type FastCache struct {
...
+
+	start := time.Now()
+	fc := &FastCache{fast: fastcache.LoadFromFileOrNew(config.FastCacheFileDir, config.LocalCacheSizeMiB*int(units.MiB))}
 	stats := fc.UpdateStats().(fastcache.Stats)
 
-	logger.Info("Initialize local trie node cache (fastCache)",
-		"MaxMB", config.LocalCacheSizeMB, "FilePath", config.FastCacheFileDir,
-		"LoadedBytes", stats.BytesSize, "LoadedEntries", stats.EntriesCount)
+	logger.Info("Initialized local trie node cache (fastCache)",
+		"LoadedMiB", stats.BytesSize/uint64(units.MiB), "LoadedEntries", stats.EntriesCount, "elapsed", time.Since(start))
```
Comment (winnie-byun) [inline storage/statedb/cache_fastcache.go:65]: Why did you add the elapsed time?? Does it take a longer time than expected?

File: storage/statedb/cache_fastcache.go [hunk 2]
```diff
@@ -41,23 +43,27 @@ type FastCache struct {
...
 	}
 
-	if config.LocalCacheSizeMB <= 0 {
+	if config.LocalCacheSizeMiB <= 0 {
 		return nil
 	}
 
-	fc := &FastCache{fast: fastcache.LoadFromFileOrNew(config.FastCacheFileDir, config.LocalCacheSizeMB*1024*1024)} // Convert MB to Byte
+	logger.Info("Initializing local trie node cache (fastCache)",
+		"MaxMiB", config.LocalCacheSizeMiB, "FilePath", config.FastCacheFileDir)
```
Comment (winnie-byun) [inline storage/statedb/cache_fastcache.go:58]: Is there a need for the new log?

**Review by winnie-byun — APPROVED** [review]

File: storage/statedb/cache_fastcache.go (hunk 2 above)
Comment (KimKyungup) [reply storage/statedb/cache_fastcache.go:58]: Yes It is. It takes more than 15 sec on my mac mini.
```
INFO[02/22,09:17:48 +09] [5] Using DeriveShaConcat!
INFO[02/22,09:17:48 +09] [47] Initializing local trie node cache (fastCache)  MaxMiB=7000 FilePath=/Volumes/Samsung_T5/cypress_migrated/datadir/fastcache
INFO[02/22,09:18:04 +09] [47] Initialized local trie node cache (fastCache)  LoadedMiB=7008 LoadedEntries=20434023 elapsed=15.862882951s
```

File: storage/statedb/cache_fastcache.go (hunk 1 above)
Comment (KimKyungup) [reply storage/statedb/cache_fastcache.go:65]: Yes It is. It takes more than 15 sec on my mac mini.

**Review by ehnuje — APPROVED** [review]
LGTM

**Review by jeongkyun-oh — APPROVED** [review]
